partialWaveFit/fitResult.h

```diff
@@ -583,7 +583,7 @@ namespace rpwa {
 			         << "cannot deduce corresponding wave name." << std::endl;
 			return "";
 		}
-		return prodAmpName(3, prodAmpName.Length() - 3);
+		return prodAmpName(3, prodAmpName.Length() - 3).Data();
 	}
```

Comment:
this would also remove the special case for `V_flat`

```
std::string
fitResult::waveNameForProdAmp(const unsigned int prodAmpIndex) const
{
	const std::string& prodAmpName = _prodAmpNames[prodAmpIndex];
	if (prodAmpName.length() == 0 or prodAmpName[0] != 'V' or prodAmpName.find('_') == std::string::npos) {
		printErr << "production amplitude name '" << prodAmpName << "' does not follow the naming convention. "
		         << "cannot deduce corresponding wave name." << std::endl;
		return "";
	}
	return prodAmpName.substr(prodAmpName.find('_')+1);
```